contractapi/contract_chaincode_test.go

```diff
@@ -438,6 +446,13 @@ func TestAddContract(t *testing.T) {
 	err = cc.addContract(mc, []string{})
 	assert.EqualError(t, err, "Multiple contracts being merged into chaincode with name customname", "should error when contract already exists with name")
 
+	// should error when no public functions
+	cc = new(ContractChaincode)
+	cc.contracts = make(map[string]contractChaincodeContract)
+	pc := new(privateContract)
+	err = cc.addContract(pc, defaultExcludes)
+	assert.EqualError(t, err, "Contracts are required to have at least 1 chaincode callable method. Contract privateContract has none", "should error when contract has no public functions")
```

Comment:
would you expect it to behave differently if the contract just had no methods in it? as opposed to just private ones?